sheepdog/blueprint/routes/views/program/__init__.py

```diff
@@ -213,6 +213,23 @@ def create_project(program):
             return flask.jsonify(trans.json)
 
 
+@utils.assert_program_exists
+def delete_program(program):
+    """
+    Delete a program given program name. If the program
+    is not empty raise an appropriate exception
+    """
+    auth.admin_auth()
+    with flask.current_app.db.session_scope() as session:
+        node = utils.lookup_program(flask.current_app.db, program)
+        if node.edges_in:
+            raise UserError('ERROR: Can not delete the program.\
+                             Program {} is not empty'.format(program))
+        session.delete(node)
+        session.commit()
+        return flask.jsonify({})
```

Comment:
may want to use `204` code for this

Reply:
Fixed at f1af871e8693867cf993353b16e7f7e26f9432eb